hbase-server/src/test/java/org/apache/hadoop/hbase/master/balancer/TestStochasticLoadBalancer.java

```diff
@@ -457,6 +450,13 @@ public void testLosingRs() throws Exception {
     assertNull(plans);
   }
 
```

Comment:
How about test that, upon _StochasticLoadBalancer.balanceCluster()_ call, verify that _DummyCostFunctions.cost()_ get indeed called? That would validate custom cost functions are indeed used.

Reply:
I was thinking about adding tests with the real cost function I want to implement, is that OK for you?

Reply:
Maybe we should keep this jira concerned only with the implementation to allow additional cost functions to be added. Then you could propose another jira with the real implementation of your cost functions, which then would also define those functions specific tests. What would you say?

Reply:
Awesome!